python/pip_install/extract_wheels/lib/wheel.py

```diff
@@ -63,9 +63,18 @@ def entry_points(self) -> Dict[str, str]:
 
             # Parse the avaialble entry points
             config = configparser.ConfigParser()
-            config.read_string(whl.read(entry_points_path).decode("utf-8"))
-            if "console_scripts" in config.sections():
-                return dict(config["console_scripts"])
+            try:
+                config.read_string(whl.read(entry_points_path).decode("utf-8"))
+                if "console_scripts" in config.sections():
+                    return dict(config["console_scripts"])
+
+            # TODO: It's unclear what to do in a situation with duplicate sections or options.
+            # For now, we treat the config file as though it contains no scripts. For more
+            # details on the config parser, see:
+            # https://docs.python.org/3.7/library/configparser.html#configparser.ConfigParser
+            # https://docs.python.org/3.7/library/configparser.html#configparser.Error
+            except configparser.Error:
+                pass
```

Comment:
I wish we could do something more in this case. We could warn, but developers are warnings-blind (for good reasons) plus this would be printed during repository rule execution which will scroll off the screen before you actually try to use the generated entry point scripts.